src/Microsoft.Diagnostics.Runtime/src/DacInterface/DacDataTargetWrapper.cs

```diff
@@ -154,6 +154,7 @@ public HResult ReadVirtual(IntPtr _, ClrDataAddress cda, IntPtr buffer, int byte
                         return HResult.E_NOTIMPL;
                     }
 
+                    Trace.TraceInformation($"ReadVirtual {address:X16} size {bytesRequested}");
```

Comment:
I probably don't want to put this in shipping code.  It would be very noisy when debugging as there's a lot of ReadVirtual calls.

Reply:
Done. That was supposed to be temporary.